crypto/blake2b/blake2b_f_fuzz_test.go

```diff
@@ -0,0 +1,75 @@
+// Only enable fuzzer on platforms with AVX enabled
+//go:build go1.7 && amd64 && !gccgo && !appengine
+// +build go1.7,amd64,!gccgo,!appengine
```

Comment:
```
//go:build go1.7 && amd64 && gc && !purego
```

> appengine is obsolete and superseded by purego, and gc is a more
> precise tag for files that use gc-syntax assembly.

Source https://cs.opensource.google/go/x/crypto/+/be400aefbc4c83e9aab51e82b8d4b12760653b47 via https://cs.opensource.google/go/x/crypto/+/master:blake2b/;bpv=1

But IMO it's fine to leave it as is, so it uses the exact same logic as the other files in this package